pkg/subnet/local.go

```diff
@@ -146,19 +153,65 @@ func (d *SubnetDeployer) doDeploy(chain string, chain_genesis string) error {
 	ctx := binutils.GetAsyncContext()
 
 	ux.Logger.PrintToUser("VM ready. Trying to boot network...")
-	info, err := cli.Start(
+	loadSnapshotOpts := []client.OpOption{
```

Comment:
Not sure I follow, are you wanting to start the snapshot in all cases?

I am not sure I agree, but I am also not sure I disagree. 

If we do this, then the subnet ID will always be the same. Also the owner of the subnet will always be the same, and finally, the number of control keys is always the same and 1.

In short, I wonder if loading via snapshot should be an optional path, or an independent path. For example, load a snapshot first, and deploy to that snapshot. Or just deploy from scratch.

I have not made up my mind about it yet but it feels like an important decision. I might be wrong though.

Reply:
Current released version has almost all of this, all beign done by ANR (same owner, same number of keys, not sure about subnet ID but it is pretty much repeated on tests). In this case we will just saving time on that same operations. I think maybe we can have this fast option as default, and give a way to deploy from scratch for the case we want to give the user more options (change the subnet owner, change the control key number, but anyway that requires new commands/arguments in the CLI and of course ANR).